.github/workflows/ufm_log_analyzer_ci_workflow.yml

```diff
@@ -53,3 +53,29 @@ jobs:
         pip install ruff==0.7.3
 
         ruff format --diff --check src/loganalyze
+
+  pytest:
+    runs-on: ubuntu-latest
```

Comment:
Please change this (for all other steps also) to ubuntu 22:04

It was wrong of me to not place a hard coded version

Reply:
done